Wordle.Api/Services/RecentStatsService.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+
+namespace Wordle.Api.Services
+{
+    public class RecentStatsService
+    {
+        public RecentStatsService(AppDbContext context)
+        {
+            _context = context;
+        }
+
+        private readonly AppDbContext _context;
+
+        public IEnumerable<RecentStats> GetRecentScoreStats(string PlayerId)
```

Comment:
Nice! New service & controller seems a bit too labor intensive for my lazy style, but this definitely should work!